src/libraries/System.Private.CoreLib/src/System/Reflection/Nullability/NullabilityInfo.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.ObjectModel;
+
+namespace System.Reflection
```

Comment:
These new classes are in the `System.Reflection` namespace, but you are putting them in `System\Reflection\Nullability` folder. I think the folder should match the namespace.

Reply:
OK, will remove the folder